storage/rocksdb/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 # TODO: Copyrights
 
+IF("${PLUGIN_ROCKSDB}" STREQUAL "NO")
+  RETURN()
+ENDIF()
+
 MACRO(SKIP_ROCKSDB_PLUGIN msg)
```

Comment:
1. there are many ways to disable a plugin, always use `IF(NOT TARGET rocksdb)`

2. wouldn't help to avoid cloning anyway, it's done in `cmake/submodules.cmake` included from the top level `CMakeLists.txt` the code you're trying to disable is a leftover from times when every submodule cloned itself.

Reply:
If you really want not to clone rocksdb, you can remove that auto-cloning code (in a pull request that we'll merge) from rocksdb **and** disable auto-cloning code in `cmake/submodules.cmake`, it's configurable:
```
git config cmake.update-submodules no
```

Reply:
Hmm. But it works and helps!

I don't want to disable globally `update-submodules` because generally I need them (f.ex. `libmariadb`).

Reply:
Btw, I already have this in my `.gitconfig`:
```
[cmake]
        update-submodules = no
```
It doesn't work.

Reply:
That's because only `submodules.cmake` checks it, not `rocksdb/CMakeLists.txt`.

But I've just removed this `git submodule update` from `rocksdb/CMakeLists.txt`, so when it'll propagate to 10.3, the problem will be gone.

Reply:
Could you please reference commit, so I could cherry-pick it.